util/apv.hexpat

```diff
@@ -185,8 +186,36 @@ struct MetadataPayload {
 
     std::print("    metadata payload[{:d}] type = {:d}({}), size = {:d}", metadata_payload_count, payloadType, ptype_str, payloadSize);
 
-    u8 payloadData[while($ < endOffset)] [[sealed]];
-
+    if (payloadType == 4) {
+        u8 itu_t_t35_country_code;
+        if (itu_t_t35_country_code == 0xFF) {
+            u8 itu_t_t35_country_code_extension;
+            u8 itu_t_t35_payload[payloadSize - 2];
+        } else {
+            u8 itu_t_t35_payload[payloadSize - 1];
+        }
+    } else if (payloadType == 5) {
+        u16 primary_chromaticity_x[0];
+        u16 primary_chromaticity_y[0];
+        u16 primary_chromaticity_x[1];
+        u16 primary_chromaticity_y[1];
+        u16 primary_chromaticity_x[2];
+        u16 primary_chromaticity_y[2];
```

Comment:
It might have an error like;
'Redeclaration of identifier' 
because the symbol names are same, maybe.
I'd like to suggest to use 'for' loop similar to APV specificiation syntax.

Reply:
Can you share test data to ensure that I reproduce this problem prior to rectifying the issue?

Reply:
I don't think it can be shared the test file because it has a license.
Instead of it, I can make a code for this metadata parsing and raise PR separately.
Please wait it.

Reply:
Thank you.

Reply:
I can now reproduce this problem and can resume work on it if that would be useful. I am also fine with waiting for a different PR.

Reply:
Please check PR;
https://github.com/AcademySoftwareFoundation/openapv/pull/130